notebooks/Notebook1.md

```diff
@@ -2,33 +2,80 @@
 
 ## Last week's critique
 
-**TODO:** Fill in this part with a summary and reflection on the critique you received for
+Fill in this part with a summary and reflection on the critique you received for
 last week's work. Answer questions such as:  How, specifically, did the feedback help
 improve the project? Did the feedback point out or offer something you hadn't considered?
 Did it help you make a design decision? Was it helpful in addressing the most pressing
 issues in your project? How will you incorporate the feedback into your work? Will you
 change something about the design, implementation, or evaluation as a result?
 
+I thought that the critique was very good for giving me clear points on what I
+should focus on and further consider when implementing my language. One example
+that the feedback pointed out which I had not considered was creating a clear
+ranking of what functionality I want to prioritize. While I know I wanted to
+prioritize some things over others, I had not created a list. I think this was
+particularly useful feedback at this stage of the project because having a
+clearer plan will allow me to ensure I am staying on track later and not falling
+into time-sink issues as we discussed in class so that what I am working on is
+crucial functionality and not unnecessary to the goals for my DSL.
+
 ## Description
 
-**TODO:** Fill in this part with information about your work this week:
+Fill in this part with information about your work this week:
 important design decisions, changes to previous decisions, open questions,
 exciting milestones, preliminary results, etc. Feel free to include images
 (e.g., a sketch of the design or a screenshot of a running program), links to
 code, and any other resources that you think will help clearly convey your
 design process.
 
+In the discussion this past Monday, I mentioned how in my research for my
+project pitch I found a numpy library which achieves a very similar goal to the
+outcome I was looking for with my DSL. So I spent much of the studio time this
+week learning how to use that library and then trying to perform different
+calculations with it.
+
+Then after having gotten experience with using the library, I commented the code
+so that I could compare the written explanation of the calculations I performed
+and the code.
+
+Finally, I reflected on this library and what features I thought were missing
+from the goals I outlined in my project pitch. I also considered two examples of
+different designs, one more crucial and one minor change which is not important
+for user needs.
+
```

Comment:
This was really good for you to do. As we discussed in class I think there is a need for such a tool but working to make it natural language based would allow you separate your language from what already exists.